File: 백준/Bronze/10810. 공 넣기/공 넣기.cc
#include <stdio.h>
int arr[101] = {0,};
int main(void) {
    int n, m;
    scanf("%d %d", &n, &m);

    int i, j, k;
    while(m--) {
        scanf("%d %d %d", &i, &j, &k);
        for (int idx = i; idx <= j; idx++) arr[idx] = k;
    }
    
    for (int idx = 1; idx <= n; idx++) printf("%d ", arr[idx]);
    return 0;
}

File: 백준/Bronze/10810. 공 넣기/공 넣기.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int m = sc.nextInt();

        int arr[] = new int[n+1];

        int i, j, k;
        for (int t = 0; t < m; t++) {
            i = sc.nextInt();
            j = sc.nextInt();
            k = sc.nextInt();

            for (int idx = i; idx <= j; idx++) arr[idx] = k;
        }

        for (int idx = 1; idx <= n; idx++) System.out.print(arr[idx] + " ");
    }
}